### ISSUE TITLE
NVDA unable to detect the activation of a split button while using track changes

### ISSUE THREAD

**Issue by pranavlal** [nvaccess/nvda#5744]
Hi,
I had a document in which track changes were enabled and I could see that there were several changes. I wanted to accept all changes. 
Note:
I am using Word 2016 on Windows 10 professional.

I had used this feature extensively in Word 2013. The steps I need to do to accept all changes are as follows.
1. Open a document in word 2016 with track changes.
2. Navigate to the review tab of the ribbon.
3. Tab to the changes group and activate it.
4. Position yourself at the "accept and move to next" split button.
5. Hit the spacebar or the enter key to activate this button.

I should get a series of options at this point but I get nothing. NVDA 2015.4 does nothing.
I subsequently contacted the Microsoft disabilities answer desk. They gave me a temporary solution where I hit alt+r to activate the review tab of the ribbon  and then the letter a to activate the split button which has the options I need like accept all changes in document, accept and move to next change etc. NVDA is able to read these options when I arrow through them. I am not sure what is happening. My method for activating this split button does not appear to work but if I remember my interaction correctly, the person from the disabilities answer desk was able to see that the options were indeed visible on the screen.

**Comment by jcsteh** [comment]
Can you please try this with a [snapshot](http://www.nvda-project.org/snapshots/) from the "master" branch? There was a recent fix related to ribbons in Microsoft Office 2013 and later which may address this.

Note that you don't need to install the snapshot if you don't wish to. You can just press "Continue running" after running the executable and agreeing to the license.

**Comment by pranavlal** [comment]
I tried nvda_snapshot_master-13227,4ea05c79.exe. I tried activating the accept changes split button without success.

**Comment by bhavyashah** [comment]
### Summary ###
* @ pranavlal, the original author of this ticket, reports that pressing the accept and move to next split button visually opens a list of options but which are not read by NVDA. This issue was not present with Word 2013 and but is applicable only to Word 2016 so far. However, there exists a work-around to this issue involving the usage of a set of keyboard shortcuts as opposed to standard navigation through ribbons and menus in Word described in https://github.com/nvaccess/nvda/issues/5744#issue-133121638.
* We have a clear STR and have narrowed down the applicability of this bug to a particular version of a software, i.e. Word 2016. We may need some more analysis about the importance of accessing this track changes related control and whether or not the work-around is adequate for and readily apparent to an average user.
* It is worth noting that there may have been updates in NVDA or Word 2016 since February of 2016 which may have ameliorated this situation. A status update from @ pranavlal on this ticket would be appreciated.

**Comment by pranavlal** [comment]
This issue is no longer occuring for me. I am on NVDA  2018.2.1. I am using Word that comes with Office 365. The issue is resolved but I do not know what fixed it.
